### PR TITLE
Dec 481 update api

### PR DESCRIPTION
**Pull request by jonhartzler**
Updates to have the api produce data in the new prescribed api format.
https://github.com/ndlib/honeycomb/wiki/Meta-Data-Api

### CODE REVIEW

**Comment by jgondron** [comment]
:+1:

File: app/decorators/v1/metadata_json.rb [hunk 1]
```diff
@@ -6,63 +6,63 @@ def self.metadata(item)
...
+    end
 
-      if value.present?
-        value
-      end
+    def date_value(value, label)
+      MetadataDate.new(value.symbolize_keys).to_hash(label)
     end
   end
 end
```
Comment (rfox2) [inline app/decorators/v1/metadata_json.rb]: Are we still using jbuilder for JSON views? I thought we were backing away from that in favor of hash constructs and to_json.

**Comment by rfox2** [comment]
Just posted a question about jbuilder. Are we still using it and writing new views with it?

File: app/decorators/v1/metadata_json.rb (hunk 1 above)
Comment (jonhartzler) [inline app/decorators/v1/metadata_json.rb]: I think the answer is no we are not.  It has never solved problems where there are complex chains very well and these are pretty complex chains.

**Comment by jaronkk** [comment]
:+1:

**Pull request merged**